helm/hpcc/README.md

```diff
@@ -0,0 +1,14 @@
+## HPCC Systems HELM Chart
+
+### Dependencies
+- HPCC Systems is designed to deploy a local, minimal Elastic Stack instance for log processing. This dependency requires a local copy of the Elastic Stack Helm charts for ElasticSearch, Filebeats and optionally Kibana. Helm provides a convenient command to pull appropriate dependancies locally:
```

Comment:
Probably better to point them instead at the --dependency-update flag, and to add it to startall.sh (perhaps optionally). That way you don't need to be in the hpcc chart location.

Reply:
Typo: dependAncies